WordPress/Resources/ja.lproj/Localizable.strings

```diff
@@ -758,13 +758,13 @@
 "Are you sure you want to mark this comment as Spam?" = "このコメントをスパムとしてマークしてもよいですか ?";
 
 /* Message asking for confirmation before marking a comment as trash */
-"Are you sure you want to mark this comment as Trash?" = "Are you sure you want to mark this comment as Trash?";
```

Comment:
Completely unrelated to this PR, but this wording of the English copy seems rude to me?
I'd expect "mark as spam" or "move to trash", but "mark as Trash" feels like a "you're trash!" insult 😅